app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -366,6 +366,7 @@ public void saveValue(Object newValue) {
         value = newValue;
     }
 
+    // FIXME?  why does this return a primitive, when the other methods return objects?  Should this return an Integer?
```

Comment:
This was likely inherited from reversed Vanced YouTube code. Using the primitive equivalent for all methods should be sufficient.

Reply:
primitive return types would be more clear, since null number values cannot be saved anyways